noir_stdlib/src/lib.nr

```diff
@@ -34,12 +34,20 @@ mod append;
 #[oracle(print)]
 unconstrained fn print_oracle<T>(with_newline: bool, input: T) {}
 
-unconstrained pub fn print<T>(input: T) {
-    print_oracle(false, input);
+unconstrained fn print_unconstrained<T>(with_newline: bool, input: T) {
+    print_oracle(with_newline, input);
```

Comment:
This was added because we can't call oracles in constrained functions, right? But when we make println and print unconstrained we'd then get `unsafe` errors in nearly every function using them. 

Making a note of this as something to keep in mind for breakage of user code since it's a somewhat different category from the other unconstrained errors.